swift.html.markdown

```diff
@@ -209,25 +209,25 @@ default: // required (in order to cover all possible input)
 // Function with Swift header docs (format as reStructedText)
 
 /**
-    A greet operation
+A greet operation
 
-    - A bullet in docs
-    - Another bullet in the docs
+- A bullet in docs
+- Another bullet in the docs
 
-    :param: name A name
-    :param: day A day
-    :returns: A string containing the name and day value.
+:param: name A name
+:param: day A day
+:returns: A string containing the name and day value.
 */
 func greet(name: String, day: String) -> String {
     return "Hello \(name), today is \(day)."
 }
-greet("Bob", "Tuesday")
+greet("Bob", day: "Tuesday")
 
 // similar to above except for the function parameter behaviors
-func greet2(#requiredName: String, externalParamName localParamName: String) -> String {
```

Comment:
Indeed, iz gone. I'm glad they removed it, felt a little weird using "#" symbol. Might be confused with the "#" for build conditions.